app/assets/javascripts/app/pages/connections/index.js.rb

```diff
@@ -80,13 +83,17 @@ def add_connection(connection_json)
       result.from.connections += 1
       result.to.connections += 1
       @connections << result
-      @points[result.team] += 1
+      update_points(result)
     }
   end
 
   def content
     add_child :network_container, Panel, title: 'Status'
     @network = network_container.add_content :network, Network, nodes: nodes, edges: edges, options: NETWORK_OPTIONS
+    points = network_container.add_to_header :points, Points
+    @point_display = TEAMS.map { |t|
```

Comment:
This can be written as `TEAMS.to_h { |t| [t, points.add_child(...)] }`.

Reply:
it seems Opal does not support this version of to_h yet

Reply:
So we have a chance to send a pull request 🚀